<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainmenu</class>
 <widget class="QWidget" name="mainmenu">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bug Tracker - Main Menu</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QStackedWidget" name="menuWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1061</width>
     <height>681</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1061</width>
     <height>681</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="loginPage">
    <widget class="QPushButton" name="loginB">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userName">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>280</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="userPassword">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>310</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="menuPage">
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <widget class="QStackedWidget" name="queryWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>811</width>
       <height>621</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="viewAllPage">
      <widget class="QTableView" name="tableViewObj">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>791</width>
         <height>601</height>
        </rect>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="searchPage">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QTableView" name="searchView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>791</width>
         <height>571</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="searchBtn">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="searchQuery">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="addPage">
      <widget class="QComboBox" name="priority">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>130</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>High</string>
        </property>
        <property name="icon">
         <iconset theme="color:Material.colo(Material.Red)">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Medium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Low</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="bugSum">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>40</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="devName">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>250</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>40</number>
       </property>
      </widget>
      <widget class="QDateEdit" name="bugDate">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="minimumDate">
        <date>
         <year>2020</year>
         <month>9</month>
         <day>1</day>
        </date>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="status">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>70</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Attention Needed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fixed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>In Progress</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="addIssueBtn">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>290</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Issue</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="bugDesc">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>120</y>
         <width>331</width>
         <height>111</height>
        </rect>
       </property>
       <property name="maximumBlockCount">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Issue Summary:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date Issue Found:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Detailed Description:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>250</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Discovered by:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Current Status</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>110</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Priority</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="viewAll">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>172</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>View Current Issues</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewIss">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Search Issues</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addIss">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Issue</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logOut">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Log out</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Tristan's Dev Tracker 1.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>1051</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
